### PR TITLE
Adding validation to the `AddProductReview` endpoint

### LINKED ISSUE #160: [Bug] Review rate validation
Any value can be pass to AddReview command, therefore any value will be accepted by AddReviewHandler

### PR DESCRIPTION
**Pull request by mamazu**
In Sylius product reviews have to have a rating between 0 and 5. Anything else is out of range. I added validation. This should also close #160.

### CODE REVIEW

**Comment by mamazu** [comment]
The reason why this fails is because the master also fails. Probably the dependencies have changes the way taxons are handled. We might want to consider running the unit tests every day to track those changes. Or what do you think @lchrusciel?

**Comment by Zales0123** [comment]
This is the same reason as in https://github.com/Sylius/ShopApiPlugin/pull/385, there must have been some changes in Symfony or in serializer :/ I don't know, to be honest, how to fix it, although I've spent some time debugging it cc @pamil @lchrusciel 🚁

File: tests/Controller/AddProductReviewBySlugTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\ShopApiPlugin\Controller\Product;
+
+use Symfony\Component\HttpFoundation\Response;
+use Tests\Sylius\ShopApiPlugin\Controller\JsonApiTestCase;
+
+final class AddProductReviewBySlugTest extends JsonApiTestCase
```
Comment (lchrusciel) [inline tests/Controller/AddProductReviewBySlugTest.php]: it should be a part of ProductAddReviewBySlugApiTest

File: tests/Controller/AddProductReviewByCodeTest.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\ShopApiPlugin\Controller\Product;
+
+use Symfony\Component\HttpFoundation\Response;
+use Tests\Sylius\ShopApiPlugin\Controller\JsonApiTestCase;
+
+final class AddProductReviewByCodeTest extends JsonApiTestCase
```
Comment (lchrusciel) [inline tests/Controller/AddProductReviewByCodeTest.php]: should be part of ProductAddReviewByCodeApiTest

**Comment by mamazu** [comment]
Fixed

File: tests/DependencyInjection/ConfigurationTest.php [hunk 3]
```diff
@@ -63,7 +63,7 @@ public function it_has_view_classes(): void
                 'page_links' => View\PageLinksView::class,
                 'payment' => View\PaymentView::class,
                 'payment_method' => View\PaymentMethodView::class,
-		'placed_order' => View\PlacedOrderView::class,
+        'placed_order' => View\PlacedOrderView::class,
```
Comment (lchrusciel) [inline tests/DependencyInjection/ConfigurationTest.php]: something still went wrong with formatting

Comment (mamazu) [reply tests/DependencyInjection/ConfigurationTest.php]: Fixed, strangely this wasn't caught by ECS.

**Comment by lchrusciel** [comment]
Thank you, @mamazu! :1st_place_medal:

**Pull request merged**